addons/controllers/testdata/test-cluster-bootstrap-2.yaml

```diff
@@ -212,3 +237,39 @@ spec:
   valuesSchema:
     openAPIv3:
       title: antrea.tanzu.vmware.com.1.2.3+vmware.1-tkg.1 values schema
+---
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: vsphere-cpi.tanzu.vmware.com.1.22.3--vmware.1-tkg.1
+  namespace: cluster-namespace-2
+spec:
+  refName: vsphere-cpi.tanzu.vmware.com
+  version: 1.22.3+vmware.1-tkg.1
+  releaseNotes: vsphere-cpi 1.22.3 https://github.com/kubernetes/cloud-provider-vsphere
+  licenses:
+    - 'VMware’s End User License Agreement (Underlying OSS license: Apache License 2.0)'
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: projects-stg.registry.vmware.com/tkg/tkgextensions-dev/packages/core/vsphere-cpi:v1.22.3_vmware.1-tkg.1
+      template:
+        - ytt:
+            paths:
+              - config/
+            ignoreUnknownComments: true
+        - kbld:
+            paths:
+              - '-'
+              - .imgpkg/images.yml
+      deploy:
+        - kapp:
+            rawOptions:
+              - --wait-timeout=30s
+              - --kube-api-qps=20
+              - --kube-api-burst=30
+  releasedAt: "2021-12-20T10:59:32Z"
+  valuesSchema:
+    openAPIv3:
+      title: vsphere-cpi.tanzu.vmware.com.1.22.3--vmware.1-tkg.1 values schema
```

Comment:
Could we also add a test case for pre-created Antrea routable config? This would cover the use case for customizing an addon config prior to creating a cluster.